8Knot/pages/repo_overview/visualizations/repo_general_info.py

```diff
@@ -31,6 +32,12 @@
                 dcc.Loading(
                     html.Div(id=f"{PAGE}-{VIZ_ID}"),
                 ),
+                dbc.Row(
+                    [
+                        dbc.Label("Last Updated:", className="mr-2"),
```

Comment:
put date next to "last updated" instead of underneath

Reply:
In theory it should be next to it. That's why I put it inside a `dbc.Row` element.

If you're just looking at my screenshots, the fonts on my browser are messed up, so that could be why it's displaying on two lines.

Reply:
![Screenshot 2024-10-08 at 5 35 21 PM](https://github.com/user-attachments/assets/cfab0df5-2438-479e-9d2a-aaf28f13cd98)
 this is what it looks like on my side

Reply:
fixed

Reply:
Can you add a space after the : ?

Reply:
spaces added